ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -437,41 +439,43 @@ try:
                                               '--type')
 
         key_pair_name = args.sshName
+        ssh_private_key_file = args.privateKey
+        if not key_name and not ssh_private_key_file and not args.accountName:
+            key_pair_name, ssh_private_key_file = setup.create_upload_key_pair()
         if not key_pair_name:
             key_pair_name = utils.get_from_config(args.accountName,
                                                   config,
                                                   region,
                                                   'ssh_key_name',
                                                   '--ssh-key-pair')
+        if not key_pair_name:
+            print('Could not determine key pair name', file=sys.stderr)
+            sys.exit(1)
 
-        ssh_private_key_file = args.privateKey
         if not ssh_private_key_file:
             ssh_private_key_file = utils.get_from_config(args.accountName,
                                                          config,
                                                          region,
                                                          'ssh_private_key',
                                                          '--private-key-file')
-        # If key_pair_name and ssh_private_key_file are missing, a temporary key pair
-        # get's created. Otherwise both parameters are required.
-        if key_pair_name and not ssh_private_key_file:
-            print('Could not determine the private ssh key file', file=sys.stderr)
-            sys.exit(1)
-        if ssh_private_key_file and not key_pair_name:
-            print('Could not determine key pair name', file=sys.stderr)
-            sys.exit(1)
 
-        if ssh_private_key_file:
-            ssh_private_key_file = os.path.expanduser(ssh_private_key_file)
+        if not ssh_private_key_file:
```

Comment:
Note that this is basically just a revert of https://github.com/SUSE/Enceladus/pull/181